patches/chrome-android-java-src-org-chromium-chrome-browser-settings-MainSettings.java.patch

```diff
@@ -1,13 +1,13 @@
-diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainPreferences.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainPreferences.java
-index 32213f862d26a7ab447fd5961d35e872eae73fa2..d4cd969199fc191dfc2762b1f6fea359a26ed45a 100644
---- a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainPreferences.java
-+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainPreferences.java
-@@ -43,7 +43,7 @@ import java.util.Map;
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
+index 0991b6499c42f846a3a80468ecc816ced513cfa9..3479ce29fdd76bfdf0019b8344fbfb0d2a5a358f 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
+@@ -42,7 +42,7 @@ import java.util.Map;
  /**
   * The main settings screen, shown when the user first opens Settings.
   */
--public class MainPreferences extends PreferenceFragmentCompat
-+public class MainPreferences extends BraveMainPreferencesBase
+-public class MainSettings extends PreferenceFragmentCompat
```

Comment:
this patch is no longer required per BraveMainPreferenceBaseClassAdapter, but the path there needs to be updated

Reply:
cc: @samartnik